hermes/src/store/proof/batch_vaa.rs

```diff
@@ -97,6 +96,18 @@ pub fn get_price_feeds_with_update_data(
 }
 
 
+pub fn get_price_feed_ids(state: State) -> Vec<PriceIdentifier> {
+    let mut price_ids = Vec::new();
+    for key in state.keys() {
+        let maybe_32_bytes: Result<[u8; 32], _> = ((*key).clone()).try_into();
```

Comment:
This really implies that the key should be an `enum`, matching on length to distinguish is an implicit/unsafe version of the same thing, so we me as well type it.